pkg/upgrade/control_plane.go

```diff
@@ -11,6 +11,8 @@ import (
 	"strings"
 	"time"
 
+	"sigs.k8s.io/yaml"
+
```

Comment:
nit: we should probably pull in those hack/verify scripts that @neolit123 has been posting around to our new sig cluster lifecycle projects. They are actually really awesome to have!

Reply:
Add an issue please?